.coveragerc

```diff
@@ -0,0 +1,31 @@
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
+
+[run]
+omit =
+     # data files
+     */test_03_core_data/*
+     # tutorials
+     *tutorials*
+     # dirs
+     */dust/*
+     */mag/*
+     tofu/openadas2tofu/*
+     tofu/imas2tofu/*
+     #scripts
+     */tofu/_*.py
```

Comment:
Why this last line ?

Scripts are in `scripts/*.py`

We can also exclude entry points :  `tofu/entrypoints/*.py`

Reply:
True for scripts, sorry, added entrypoints !